Ruby 2.4 rubyio.h warning

Compiling terminfo.c today raised the following warning:

    /usr/include/ruby-2.4.0/ruby/backward/rubyio.h:2:2: warning: #warning use "ruby/io.h" instead of "rubyio.h" [-Wcpp]

I think it should be as simple as updating [line 33](https://github.com/genki/ruby-terminfo/blob/master/terminfo.c#L33)? I don't have the time right now, but I can test later this week if you don't get to first.

--- a/Gemfile
+++ b/Gemfile
@@ -1,0 +1,3 @@
+source 'http://rubygems.org'
+
+gemspec

--- a/extconf.rb
+++ b/extconf.rb
@@ -37,9 +37,9 @@ require 'mkmf'
 have_library("ncurses", "setupterm") or
 have_library("curses", "setupterm") 
 
-have_type("rb_io_t", ["ruby.h", "rubyio.h"])
-have_struct_member("rb_io_t", "fd", ["ruby.h", "rubyio.h"])
-have_struct_member("OpenFile", "fd", ["ruby.h", "rubyio.h"])
+have_type("rb_io_t", ["ruby.h", "ruby/io.h"])
+have_struct_member("rb_io_t", "fd", ["ruby.h", "ruby/io.h"])
+have_struct_member("OpenFile", "fd", ["ruby.h", "ruby/io.h"])
 
 create_header
 create_makefile('terminfo')
@@ -50,4 +50,3 @@ rdoc:
 	rdoc --op rdoc terminfo.c lib/terminfo.rb
 End
 }
-

--- a/terminfo.c
+++ b/terminfo.c
@@ -30,7 +30,7 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 */
 
 #include "ruby.h"
-#include "rubyio.h"
+#include "ruby/io.h"
 #include "extconf.h"
 
 #include <curses.h>

